packages/typescript/internal/ts_project.bzl

```diff
@@ -143,6 +143,7 @@ def _ts_project_impl(ctx):
     # Improves error messaging if a ts_project needs declaration = True
     if len(typings_outputs) or len(ctx.attr.deps):
         providers.append(declaration_info(depset(typings_outputs), ctx.attr.deps))
+        providers.append(OutputGroupInfo(type_defs = depset(typings_outputs)))
```

Comment:
bike-shedding a bit, because we'll never be able to change this name - "type_defs" doesn't seem like a term we have in TypeScript or in the bazel rules right now. should it be "typings" or "types"?

also could you add a `filegroup` somewhere under `test/ts_project/*/BUILD.bazel` that exercises this and demonstrates how it's used?

Reply:
yup definitely thanks!